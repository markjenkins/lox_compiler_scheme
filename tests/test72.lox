// from craftinginterpreters/test/precedence.lox

// > has higher precedence than ==.
print false == 1 > 2; // expect: true
