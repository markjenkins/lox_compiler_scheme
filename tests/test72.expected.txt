OP_FALSE
OP_CONSTANT 1
OP_CONSTANT 2
OP_GREATER
OP_EQUAL
OP_PRINT
OP_RETURN
